docs/docs/pods.md

```diff
@@ -0,0 +1,583 @@
+  "scaling": { "kind": "fixed", "instances": 1 },
+  "containers": [
+    {
+      "name": "sleep1",
+      "exec": { "command": { "shell": "sleep 1000" } },
+      "resources": { "cpus": 0.1, "mem": 32 },
+      "endpoints": [ { "name": "web", "containerPort": 80, "protocol": [ "http" ] } ],
+      "healthCheck": { "http": { "endpoint": "web", "path": "/ping" } }
+    }
+  ],
+  "networks": [ { "mode": "container", "name": "my-virtual-network-name" } ]
+}
+```
+
+## Complete Pod 
+The following pod definition can serve as a reference to create more complicated pods.
+
+```
+{
+  "id": "/complete-pod",
+  "labels": {
+    "owner": "zeus",
+    "note": "Away from olympus"
+  },
+  "environment": {
+    "XPS1": "Test"
+  },
+  "volumes": [
+    {
+      "name": "etc",
+      "host": "/hdd/tools/docker/registry"
+    }
+  ],
+  "networks": [
+    {
+      "mode": "host"
+    }
+  ],
+  "scaling": {
+    "kind": "fixed",
+    "instances": 1
+  },
+  "scheduling": {
+    "backoff": {
+      "backoff": 1,
+      "backoffFactor": 1.15,
+      "maxLaunchDelay": 3600
+    },
+    "upgrade": {
+      "minimumHealthCapacity": 1,
+      "maximumOverCapacity": 1
+    },
+    "placement": {
+      "constraints": [],
+      "acceptedResourceRoles": []
+    }
+  },
+  "containers": [
+    {
+      "name": "container1",
+      "exec": {
+        "command": {
+          "shell": "sleep 100"
+        },
+        "overrideEntrypoint": false
+      },
+      "resources": {
+        "cpus": 1,
+        "mem": 128,
+        "disk": 0,
+        "gpus": 0
+      },
+      "endpoints": [
+        {
+          "name": "http-endpoint",
+          "containerPort": 80,
+          "hostPort": 0,
+          "protocol": [ "http" ],
+          "labels": {}
+        }
+      ],
+      "image": {
+        "id": "mesosphere/marathon:latest",
+        "kind": "DOCKER",
+        "forcePull": false
+      },
+      "environment": {
+        "XPS1": "Test"
+      },
+      "user": "root",
+      "healthCheck": {
+        "gracePeriodSeconds": 30,
+        "intervalSeconds": 2,
+        "maxConsecutiveFailures": 3,
+        "timeoutSeconds": 20,
+        "delaySeconds": 2,
+        "http": {
+          "path": "/health",
+          "scheme": "HTTP",
+          "endpoint": "http-endpoint"
+        }
+      },
+      "volumeMounts": [
+        {
+          "name": "env",
+          "mountPath": "/mnt/etc",
+          "readOnly": true
+        }
+      ],
+      "artifacts": [
+        {
+          "uri": "https://foo.com/archive.zip",
+          "executable": false,
+          "extract": true,
+          "cache": true,
+          "destPath": "newname.zip"
+        }
+      ],
+      "labels": {
+        "owner": "zeus",
+        "note": "Away from olympus"
+      },
+      "lifecycle": {
+        "killGracePeriodSeconds": 60
+      }
+    }
+  ]
+}
+```
+
+# Limitations
+
+- If a pod belongs to a group that declares dependencies, these dependencies are implicit for the pod.
```

Comment:
I don't know exactly what this is meaning... are you saying that a dependency in an app or container means that the pod has that dependency?  (seems obvious) but perhaps you mean something else.